src/App/App.tsx

```diff
@@ -1,7 +1,62 @@
-import React from "react";
+import Comment from "@/components/Comment";
+import CommentsSection from "@/components/CommentsSection";
+import Layout from "@/components/Layout";
+import { useEffect, useState } from "react";
+interface Data {
+  comments: Comment[];
+  currentUser: {
+    image: {
+      png: string;
+      webp: string;
+    };
+    username: string;
+  };
+}
+
+type Reply = Comment & { replyingTo: string };
+
+interface Comment {
+  id: number;
+  content: string;
+  createdAt: string;
+  score: number;
+  user: {
+    image: {
+      png: string;
+      webp: string;
+    };
+    username: string;
+  };
+  replies: Reply[] | [];
+}
 
 const App = () => {
-  return <div>d</div>;
+  const [data, setData] = useState<Data>();
+
+  useEffect(() => {
+    fetch("../../data.json")
+      .then((res) => res.json())
+      .then((data) => setData(data));
+  }, [data]);
+
+  return (
+    <Layout>
+      <CommentsSection>
+        {data &&
+          data.comments.map((comment) => {
```

Comment:
C optional chaining красивше
```suggestion
          data?.comments.map((comment) => {
```